.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Daily build
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
```suggestion
  push:
    branches:
      - master
```

Reply:
Why do we want to release the docker image all the time that we marge into main? 
What would happen if two PRs are merged at the same time?

Reply:
This is only to make development easier for me, will remove after I am done